src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_IndexerAccess.cs

```diff
@@ -312,7 +398,6 @@ private BoundSequence VisitIndexOrRangePatternIndexerAccess(BoundIndexOrRangePat
             var sideEffectsBuilder = ArrayBuilder<BoundExpression>.GetInstance();
 
             var receiverLocal = F.StoreToTemp(VisitExpression(receiver), out var receiverStore);
```

Comment:
It looks like the comment on 387 is no longer accurate.

Reply:
> It looks like the comment on 387 is no longer accurate.

Are you suggesting swapping the following lines?
```
            // int length = receiver.length;
            // Range range = argumentExpr;
```

Reply:
Yes, it looks like that's all that would need to change to make the comment accurate.